packages/core/demo/demo-options.ts

```diff
@@ -1,46 +1,55 @@
 import { colorPalettes } from "../src/index";
 
 export const setOrUpdateParam = (name, value) => {
```

Comment:
can't revert the formatting on demo-options because it causes a tslint error (_tab indentation expected (indent)_) and doesnt allow you to push unless tabs are used.